src/peft/tuners/poly/config.py

```diff
@@ -0,0 +1,57 @@
+from dataclasses import dataclass, field
+from typing import List, Optional, Union
+
+from peft.config import PeftConfig
+from peft.utils import PeftType
+
+
+@dataclass
+class PolyConfig(PeftConfig):
```

Comment:
Let's add a docstring. This would also be a good place to reference the paper(s).

Reply:
done!